app/routes/labs.missions/route.tsx

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { GridContainer } from '@trussworks/react-uswds'
+
+import fermiHeroImage from './Fermi_Earth_GWs.jpg'
+import { Meatball } from '~/components/meatball/Meatball'
+import type { BreadcrumbHandle } from '~/root/Title'
+
+import fleet from './AstroFleetChart.png'
+
+export const handle: BreadcrumbHandle = {
+  breadcrumb: 'Missions',
+}
+
+export default function () {
+  return (
+    <>
+      <img
+        alt="Fermi satellite with a GRB in the background, both positioned over earth"
```

Comment:
Add `width` and `height` attributes to this and other images to prevent layout jumps while loading.

Reply:
I've done a mobile display-block that fixes it.

Reply:
the issue is that if you just hard code a size for the image, on mobile it looks terrible. so I currently am setting it to display-block when it's either mobile or tablet.

Reply:
Right, so what I do is I set the width and height to the actual pixel size of the image, and then I use utility classes like `"width-5 height-auto"` to size it for the page.

Reply:
ok, let me try that